api-test.json

```diff
@@ -50514,16 +50453,32 @@
 													"response": []
 												},
 												{
-													"name": "Delete entity",
+													"name": "Create partial matching reg simple Entity",
 													"event": [
 														{
 															"listen": "test",
 															"script": {
 																"exec": [
-																	"\r",
-																	"pm.test(\"Status code is 204\", function () {\r",
-																	"    pm.response.to.have.status(204);\r",
+																	"var payloadBody = JSON.parse(pm.request.body.raw);\r",
+																	"console.log(payloadBody.id);\r",
+																	"var entityId = payloadBody.id;\r",
+																	"pm.collectionVariables.set(\"created\", payloadBody);\r",
+																	"pm.collectionVariables.set(\"entityId\", entityId);\r",
+																	"pm.test(\"Status code is 207\", function () {\r",
+																	"    pm.response.to.have.status(207);\r",
+																	"});\r",
+																	"pm.test(\"Check location type and title\", function () {\r",
+																	"    var jsonData = pm.response.json();\r",
+																	"    var success = jsonData[\"success\"];\r",
+																	"    var fails = jsonData[\"failure\"];\r",
+																	"    var regFail = fails[0];\r",
+																	"    \r",
+																	"    pm.expect(regFail[\"type\"]).to.equal(\"https://uri.etsi.org/ngsi-ld/errors/UnprocessableContextSourceRegistration\");\r",
+																	"    pm.expect(regFail[\"title\"]).to.equal(\"UnprocessableContextSourceRegistration\");\r",
+																	"    pm.expect(success[0][\"attributes\"][0][\"attributeName\"]).to.equal(\"notBrandName\");\r",
 																	"});\r",
+																	"\r",
```

Comment:
Add test case for check entity type(/types/{entityType} API)

Reply:
added